components/feature/searchwidget/src/main/res/xml/mozac_search_widget_info.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Don't think we need this in AC?
AC can be left to handle the system broadcasts about the widget but the widget configuration - done in xml will be left for each client to take care of.
This will also mean AC doesn't need to offer that `previewImage` (which now has the Fenix icon).

Reply:
Done